test/regress/regress1/bug520.smt2

```diff
@@ -1,7 +1,8 @@
 ; COMMAND-LINE: --bitblast=lazy
-; COMMAND-LINE: --bitblast=eager --no-check-models
+; COMMAND-LINE: --bitblast=eager
 ; COMMAND-LINE: --bv-solver=bitblast-internal
 ; EXPECT: sat
+; DISABLE-TESTER: model
```

Comment:
It may be a bit ugly, but what do you think about having two copies of this test (one with command line options 1 and 3 with the model tester and one with command line options 2 without the model tester)?

Reply:
Good catch, I meant to fix this. Ordinarily, I would split this benchmark but its kind of large.  Another option would be to allow to use `--no-check-models` in this one case.